.github/workflows/test-common.yml

```diff
@@ -65,7 +48,19 @@ jobs:
           set -x # print commands that are executed
           coverage erase
 
-          ./scripts/runtox.sh "py${{ matrix.python-version }}$" --cov=tests --cov=sentry_sdk --cov-report= --cov-branch --ignore=tests/integrations
+          ./scripts/runtox.sh "py${{ matrix.python-version }}-${{ matrix.variant }}$" --cov=tests --cov=sentry_sdk --cov-report= --cov-branch --ignore=tests/integrations
           coverage combine .coverage*
           coverage xml -i
           codecov --file coverage.xml
+
+  check_required_tests:
+    name: All common tests passed or skipped
```

Comment:
I think we do not need the `variant` here and the `check_required_tests` step.

If we just keep your changes in `tox.ini` and then you run locally `python scripts/split-tox-gh-actions/split-tox-gh-actions.py ` then a new github workflow file `test-gevent.yml` will be created that will run all tests in the specified python version and with the specified `gevent` requirement.

Reply:
You are right. But you can set the path where to find the tests here: https://github.com/getsentry/sentry-python/blob/master/tox.ini#L377-L379

So I took the freedom to make some changes to your PR to run all the tests with gevent like the other integration tests.

Reply:
resolve because it is outdated now.